internship/src/calculator/Calculator.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+
+
+
+class Calculator extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            firstNumber: '',
+            secondNumber: '',
+            sign: '',
+            result: '',
+        };
+        this.changeFirstNumber = this.changeFirstNumber.bind(this);
+        this.changeSecondNumber = this.changeSecondNumber.bind(this);
+        this.changeSign = this.changeSign.bind(this);
+        this.calculate = this.calculate.bind(this);
+    };
+
+
+    render() {
+        return(
+        <form>
+            <input type="number" value={this.state.firstNumber} onChange={this.changeFirstNumber}/>
+            <input value={this.state.sign} onChange={this.changeSign}/>
+            <input type="number" value={this.state.secondNumber} onChange={this.changeSecondNumber}/>
+            <button type="button" onClick={this.calculate}>=</button>
+            <input type="text" value={this.state.result} disabled="disabled"/>
+        </form>
+        );
+    };
+
+    changeFirstNumber(e) {
+        this.setState({firstNumber: e.target.value});
+    }
+
+    changeSecondNumber(e) {
+        this.setState({secondNumber: e.target.value});
+    }
+
+    changeSign(e) {
+        this.setState({sign: e.target.value});
+    }
+
+    calculate(e) {
```

Comment:
okay, now try to add this method for your empty file export this function a import in Calculator class.

function Square(props) {
returm (

{props.value}

);
}
And try to export default Calculate. I'm not pretty sure for this solution. You have to try.